### PR TITLE
UG add Known Issues and FAQ

### LINKED ISSUE #301: UG: Update FAQ and Known Issues
image

Poor thing so blank :(

### PR DESCRIPTION
**Pull request by raydenlim**

### CODE REVIEW

**Review by ldinghan — APPROVED** [review]
Great addition! LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1746,9 +1785,52 @@ Format: `deleteconsult CONSULTATION_INDEX`
...
+
+<box light>
+<md>
+
+**Issue:** I cannot add other email domains to the list. <br>
+* Emails are only limited to NUS emails.
 
-1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
+**Fix:** 
+* Unfortunately, as of the current of F.A.K.E.J.A.R.V.I.S. we only support NUS emails ending with `@u.nus.edu`.
```
Comment (ldinghan) [inline docs/UserGuide.md]: as of the current version of

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1746,9 +1785,52 @@ Format: `deleteconsult CONSULTATION_INDEX`
...
+</box>
+
+<box light>
+<md>
+
+**Issue:** I cannot add Names, Tags and TelegramHandles with special characters. <br>
+* Names, Tags and TelegramHandles are not able to accept special characters (e.g `Spencer O'Brian`, `Mohan S/O Rohan`, `façade`, `bob.the.best123`).<br>
+
+**Fix:** 
+* As a workaround, you may omit the special characters and input just the alphanumerics.
```
Comment (ldinghan) [inline docs/UserGuide.md]: alphanumeric characters

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1746,9 +1785,52 @@ Format: `deleteconsult CONSULTATION_INDEX`
...
+## Known Issues
+
+<box light>
+<md>
+
+**Issue:** I cannot access F.A.K.E.J.A.R.V.I.S. after using multiple screens on my device. <br>
+* If you move the F.A.K.E.J.A.R.V.I.S. to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen.<br>
+
+**Fix:** 
+* To remedy this issue, simply delete the `preferences.json` file created by F.A.K.E.J.A.R.V.I.S. before running the application again.
```
Comment (ldinghan) [inline docs/UserGuide.md]: To solve this issue

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1736,8 +1736,47 @@ Format: `deleteconsult CONSULTATION_INDEX`
...
+</md>
+</box>
+
+<box type="info" seamless icon=":fas-question:">
+
+<md>
+
+**QNS**: How do I reset all the grades of my student?<br>
+
+**ANS**: You can use the `edit` command and simply input `edit STUDENT_INDEX gt/default`, and all the scores will be set to `-`.
```
Comment (ldinghan) [inline docs/UserGuide.md]: use the `edit` command and enter the following

**Review by ldinghan — APPROVED** [review]
Some nits but LGTM

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1736,8 +1736,47 @@ Format: `deleteconsult CONSULTATION_INDEX`
...
 
-**Q**: How do I transfer my data to another computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous F.A.K.E.J.A.R.V.I.S. home folder.
+<box type="info" seamless icon=":fas-question:">
+
+<md>
+
+**QNS**: How do I transfer my data to another computer?<br>
+
+**ANS**: You can install the app in the other computer and replace the sample data files with the data files in your previous F.A.K.E.J.A.R.V.I.S. folder.
```
Comment (wesho1107) [inline docs/UserGuide.md]: I'm not sure if "QNS" and "ANS" are fit for use here as readers may be easily confused between the two as to how they look similar.

**Review by wesho1107 — APPROVED** [review]
Other than a few nits, LGTM!

File: docs/UserGuide.md (hunk 5 above)
Comment (raydenlim) [reply docs/UserGuide.md]: ok, will change it back to "Q" and "A"

**Pull request merged**